cmd/xl-v1-multipart.go

```diff
@@ -627,8 +627,10 @@ func (xl xlObjects) PutObjectPart(bucket, object, uploadID string, partID int, s
 		}
 	}
 
+	allowEmptyErasure := true
```

Comment:
allowEmpty is fine.. 'erasureCreateFile' is already the function indicating 'erasure' we don't need to redundantly say it again.